### DISCUSSION TITLE
Configure Lemminx to be enabled for XSLT

### DISCUSSION THREAD

**Discussion by muhmud** [williamboman/nvim-lsp-installer#431] [Q&A] [answered]
How can I configure lemminx so that it's enabled whenever I open an XSLT file?

I'm using [lunarvim](https://github.com/LunarVim/LunarVim), and whenever I open an XSLT file, LSP is inactive.

**Comment by williamboman** [comment, marked as answer]
Hello! So it seems like the [default filetypes](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/server_configurations/lemminx.lua#L6) for Lemminx doesn't include XSLT. I'm actually not sure what filetype XSLT generally has in (neo)vim, but I'd wager it's `xslt`. You can override the filetypes yourself when setting up the server (I can't recall how this is done in LunarVim, but it's in their docs I believe).

If Lemminx supports the `xslt` filetype, I think the best solution would be to actually upstream that change to lspconfig itself.

**Reply by muhmud** [reply]
Thanks for getting back to me; yep, I think I'll create a PR for the change in lspconfig.

The extension is commonly `xsl`, however, `xslt` gets used as well.
